cmd/kubeadm/app/cmd/join.go

```diff
@@ -327,7 +327,7 @@ func newJoinData(cmd *cobra.Command, args []string, opt *joinOptions, out io.Wri
 		opt.externalcfg.Discovery.BootstrapToken = nil
 	} else {
 		if len(opt.cfgPath) == 0 && len(args) > 1 {
-			klog.Warningf("[preflight] WARNING: More than one API server endpoint supplied on command line %v. Using the first one.", args)
+			klog.Warningf("[preflight] WARNING: More than one API server endpoint supplied on command line %v. Using the first one", args)
```

Comment:
`WARNING: More than one API server endpoint supplied on command line %v, using the first argument.`